diff --git a/src/gui/frame.py b/src/gui/frame.py
index 8afa18a..c442fe7 100644
--- a/src/gui/frame.py
+++ b/src/gui/frame.py
@@ -24,8 +24,8 @@ def resource_path(filename):
     if os.getenv("TTR_LAUNCHER_RESOURCES") is not None:
         print(os.path.join(os.getenv("TTR_LAUNCHER_RESOURCES"), filename))
         return os.path.join(os.getenv("TTR_LAUNCHER_RESOURCES"), filename)
-    print(os.path.join("/", "app", "share", "ttassets", filename))
-    return os.path.join("/", "app", "share", "ttassets", filename) #A default path
+    print(os.path.join("/", "usr", "share", "ttassets", filename))
+    return os.path.join("/", "usr", "share", "ttassets", filename) #A default path
 
 #TODO: Investigate why this doesn't work when running via flatpak
 class WebEnginePage(QWebEnginePage):
