diff --git a/toontown b/toontown
index 1bab7d2..7d4f5b3 100755
--- a/toontown
+++ b/toontown
@@ -23,7 +23,7 @@ from shutil import copyfile
 #By default, the fullscreen mode can span both monitors on Linux and it can be a hassle to fix it.
 if not os.path.exists("settings.json"):
     print("Installing default config...")
-    copyfile("/app/share/ttassets/resources/settings.json", "settings.json")
+    copyfile("/usr/share/ttassets/resources/settings.json", "settings.json")
 
 #Basically the exact same as the old launcher
 class Process(multiprocessing.Process):
