diff --git a/src/launcher/launcher.py b/src/launcher/launcher.py
index 9bf15dc..c43ff1b 100644
--- a/src/launcher/launcher.py
+++ b/src/launcher/launcher.py
@@ -252,7 +252,7 @@ class TTRLauncher(FSM):
         self.patcher = threading.Thread(target=Patcher.Patch, name='Patcher-Thread', args=(self.__updateProgress, self.__updateFile))
         self.patcher.daemon = True
         self.patcher.start()
-        while self.patcher.isAlive():
+        while self.patcher.is_alive():
             time.sleep(0.2)
 
         self.request('GetCredentials')
